site/client/owid.scss

```diff
@@ -790,10 +790,12 @@ div.owid-presentations li:nth-of-type(1) a:hover {
 }
 
 .with-sidebar .section-heading {
-    position: relative;
-    width: calc(100vw - #{$sidebar-closed-drawer-width});
-    left: $sidebar-closed-drawer-width;
-    margin-left: calc(50% - 50vw - #{$sidebar-closed-drawer-width / 2});
+    @include md-up {
```

Comment:
nit: not sure what 'md-up' means. Is that medium size and larger?

Reply:
yep!